tableschema/types/array.py

```diff
@@ -4,42 +4,21 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import six
 import json
-from .. import exceptions
-from .. import helpers
-from . import base
+from ..config import ERROR
 
 
 # Module API
 
-class ArrayType(base.JTSType):
-
-    # Public
-
-    name = 'array'
-    null_values = helpers.NULL_VALUES
-    supported_constraints = [
-        'required',
-        'pattern',
-        'enum',
-        'minLength',
-        'maxLength',
-    ]
-    # ---
-    python_type = list
-
-    def cast_default(self, value, fmt=None):
-
-        if isinstance(value, self.python_type):
-            return value
-
+def cast_array(format, value):
```

Comment:
Not sure if there is a previous context on this but wouldn't it make more sense to move `format` to an optional keyword argument that defaults to `default` as this is what people will need 90% of the time: `cast_*(value)`. Others can use `cast_string(value, format='email')` or just `cast_string(value, 'email')`.